scalafmt-tests/src/test/resources/scala3/Macro.stat

```diff
@@ -31,7 +31,13 @@ throw new AssertionError(s"failed assertion: ${${ showExpr(expr) }}")
 '{   (x: T) => ${ f('x) } }
 >>>
 '{ (x: T) => ${ f('x) } }
-<<< unctions spliced
+<<< functions with RedundantBraces
+rewrite.rules = [RedundantBraces]
+===
+'{   (x: T) => ${ f('x) } }
```

Comment:
one other thing: can you create another test case, with extra braces `${ { f('x) } }`?